docs/en/05_Releases_and_changelogs/cwp_2.6.0.md

```diff
@@ -90,11 +102,10 @@ In order to update an existing site to use the new CWP recipe the following chan
     "silverstripe/recipe-reporting-tools": "1.6.0@stable",
     "cwp/cwp-recipe-search": "2.6.0@stable",
     "silverstripe/recipe-services": "1.6.0@stable",
-
-    TODO: confirm new version (below are the old versions, which may still be correct)
-    "tractorcow/silverstripe-fluent": "4.4.5@stable",
+    "silverstripe/subsites": "2.3.3@stable",
```

Comment:
We specifically call out in [this changelog](https://github.com/silverstripe/cwp/pull/285/files#diff-b3c41ce170a6e31de504c72f08c16de1R79-R80) that we won't be including `silverstripe/subsites` in the upgrade guide anymore, which is why it wasn't present already 👍

Reply:
fair enough. gonna remove it then